spring-cloud-app-starter-metadata-maven-plugin/src/main/test/java/org/springframework/cloud/dataflow/completion/Expresso.java

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright 2018 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.springframework.cloud.dataflow.completion;
+
+/**
+ * @author David Turanski
+ **/
+public enum Expresso {
```

Comment:
Do you mean `espresso` maybe ? 😄

Reply:
Haha I borrowed this from dataflow tests, named to be similar to "expression".